client/components/navbar/navbar.service.js

```diff
@@ -332,6 +332,7 @@ class ManagerNavbarService {
 
   getAssistanceMenu({ ovhSubsidiary: subsidiary }) {
     const mustDisplayNewMenu = ['FR'].includes(subsidiary);
+    const mustDisplayChatbot = ['FR'].includes(subsidiary);
```

Comment:
the list of subsidiaries could be a constant

Reply:
Why is the variable right above still called `new` ? 🤔